js/private/test/shellcheck_launcher.sh

```diff
@@ -338,7 +338,7 @@ fi
 
 # Gather node options
 NODE_OPTIONS=()
-
+NODE_OPTIONS+=("--preserve-symlinks-main")
```

Comment:
is there also a cmd.exe change?

Reply:
These is no cmd.exe. We carry a bash depedency on Windows.